app/assets/stylesheets/layout/navbar.scss

```diff
@@ -177,6 +181,12 @@ header#main_header {
     }
   }
 
+  .navbar-inverse, .navbar-toggle:hover {
+    margin-left:5px;
+    margin-top:5px;
```

Comment:
great, thanks!  I couldn't see margins changing in chrome dev tools when i hovered/moved away, so I couldn't identify what was causing it...